test/api/ShortcutSearch.js

```diff
@@ -0,0 +1,270 @@
+process.env.NODE_ENV = 'test';
+
+const User = require('../../api/main_endpoints/models/User.js');
+
+// Require the dev-dependencies
+const chai = require('chai');
+const mongoose = require('mongoose');
+
+const chaiHttp = require('chai-http');
+const {
+  OK,
+  UNAUTHORIZED,
+  FORBIDDEN
+} = require('../../api/util/constants').STATUS_CODES;
+const SceApiTester = require('../util/tools/SceApiTester');
+const sinon = require('sinon');
+
+let app = null;
+let test = null;
+
+const expect = chai.expect;
+const tools = require('../util/tools/tools.js');
+const {
+  setTokenStatus,
+  resetTokenMock,
+  restoreTokenMock,
+  initializeTokenMock
+} = require('../util/mocks/TokenValidFunctions');
+
+const {
+  setDiscordAPIStatus,
+  resetDiscordAPIMock,
+  restoreDiscordAPIMock,
+  initializeDiscordAPIMock
+} = require('../util/mocks/DiscordApiFunction');
+const { MEMBERSHIP_STATE } = require('../../api/util/constants');
+const { getMemberExpirationDate } = require('../../api/main_endpoints/util/userHelpers.js');
+
+chai.should();
+chai.use(chaiHttp);
+
+// Our parent block
+describe('ShortcutSearch', () => {
+  before(async () => {
+    initializeTokenMock();
+    initializeDiscordAPIMock();
+    app = tools.initializeServer([
+      __dirname + '/../../api/main_endpoints/routes/ShortcutSearch.js',
+    ]);
+    test = new SceApiTester(app);
+    // Before each test we empty the database
+    await tools.emptySchema(User);
+    const testUser = new User({
+      email: 'a@b.c',
+      password: 'Passw0rd',
+      firstName: 'firstName',
+      lastName: 'lastName',
+      major: 'Software Engineering',
+    });
+    testUser.save();
+  });
+
+  after(done => {
+    restoreTokenMock();
+    restoreDiscordAPIMock();
+    tools.terminateServer(done);
+  });
+
+  beforeEach(() => {
+    setTokenStatus(false);
+    setDiscordAPIStatus(false);
+  });
+
+  afterEach(() => {
+    resetTokenMock();
+    resetDiscordAPIMock();
+  });
+
+  const token = '';
+
+  describe('POST /search', () => {
```

Comment:
also update the associated lines in the test
```suggestion
  describe('POST /', () => {
```